src/meta/test/meta_app_operation_test.cpp

```diff
@@ -114,32 +119,73 @@ TEST_F(meta_app_operation_test, create_app)
     // - create succeed with success_if_exist=true
```

Comment:
It will be better to add the unit cases this pr adds in the case comments.

Reply:
Yes, I think this is necessary since this PR introduces lots of cases.